airtime_mvc/application/common/PodcastManager.php

```diff
@@ -64,7 +64,7 @@ protected static function _findUningestedEpisodes($podcast, $service) {
             //  episodes in the list of episodes to ingest, don't skip this episode - we should try to ingest the
             //  most recent episode when the user first sets the podcast to automatic ingest.
             // If the publication date of this episode is before the ingest timestamp, we don't need to ingest it
-            if ((empty($ts) && !empty($episodes)) || strtotime($episodeData["pub_date"]) < strtotime($ts)) {
```

Comment:
Publication dates have to comply with [RFC-822](https://www.ietf.org/rfc/rfc0822.txt), so they have to include a timestamp (see also [PHP's RFC-822 format string](http://php.net/manual/en/class.datetime.php#datetime.constants.rfc822) strtotime converts the timestamps to Unix format, so the comparison should be timezone agnostic (to be consistent, the timestamps are stored after being passed through gmdate).